src/OneExplorer.ts

```diff
@@ -123,7 +123,18 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
   readonly onDidChangeTreeData: vscode.Event<OneNode|undefined|void> =
       this._onDidChangeTreeData.event;
 
-  constructor(private workspaceRoot: vscode.Uri|undefined) {}
+  // TODO(dayo) Get the ext list(cfg,tflite..) from backend
+  private fileWatcher =
+      vscode.workspace.createFileSystemWatcher(`**/*.{cfg,tflite,onnx,circle,tvn}`);
+
+  constructor(private workspaceRoot: vscode.Uri|undefined) {
+    const fileWatchersEvents =
+        [this.fileWatcher.onDidCreate, this.fileWatcher.onDidChange, this.fileWatcher.onDidDelete];
```

Comment:
`onDidChange` is need for auto refresh?
I thought that `onDidCreate` and `onDidDelete` are enough @.@
(I can be wrong...)

Reply:
Sure, it's critical. It will reflect immediate change on .cfg file. For example you change `input_path=add.tflite` to `input_path=sub.tflite` it will show the difference immediately.

Reply:
When user changes the cfg `input_path` data, this logic will move the cfg file path to another base model. I think in that case, it would be good to give users a notification that the cfg location will be changed. Without noti, it may be confusing that the file position suddenly changes.

Reply:
@jyoungyun Thanks for the opinion. 😄

Reply:
https://github.com/Samsung/ONE-vscode/issues/598#issuecomment-1125622234